### ISSUE TITLE
Custom builds with codec support?

### ISSUE THREAD

**Issue by AndrewDubya** [umihico/docker-selenium-lambda#122]
Hi,

I'm curious where the chrome build is sourced from. I'm running into an error when loading video metadata from JavaScript, and I'm not sure but I suspect this container has a chromium build without codecs.

[0923/204454.046647:INFO:CONSOLE(224)] "Error [object Event]", source: ... (224)
[0923/204454.766446:WARNING:spdy_session.cc(3234)] Received RST for invalid stream1
[0923/204455.008491:INFO:CONSOLE(54)] "Media Error {"isTrusted":true}", source: ... (54)
[0923/204455.008850:INFO:CONSOLE(54)] "Handler promise(s) failed {"msg":"Error loading requested src {\"isTrusted\":true}"}", source: ... (54)

Is it possible to use google-chrome itself, or are there other builds with codec support available?

Thanks for creating this repository! I've been trying to find a working version of Chrome for lambda for days, and this helped me so much!

**Comment by umihico** [comment]
I'm not so aware about diffs between google-chrome and chromium.

My Dockerfile downloads from [here](https://commondatastorage.googleapis.com/chromium-browser-snapshots/index.html?prefix=Linux_x64/1027016/) and we don't have other options as you see.

AWS Lambda may have strange behavior if you(r chrome) try to use GPU. Maybe is this why? My code must work not only in AWS Lambda. Could you try same thing in EC2?

**Comment by AndrewDubya** [comment]
Hi,
Thanks for the response! The default chromium builds only include open source codecs, which is why I was having that problem.

This is a *very* dirty hack, but I managed to get it working by copying just the chrome binary from the "Ungoogled Chrome" project into your chromium build.
https://ungoogled-software.github.io/ungoogled-chromium-binaries/releases/linux_portable/64bit/

I disabled the gpu via flags.

I'm surprised it worked, and I need to find a better solution, but I'm ok with it for the time being. Thanks again!
